cmake/Futility_Configurations.cmake

```diff
@@ -276,6 +277,11 @@ ELSEIF(CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
     SET(Fortran_FLAGS_RELEASE
         ${CSYM}O3
        )
+
```

Comment:
What about other compilers... could you just add ${Fortran_FLAGS_DEBUG} ${Fortran_FLAGS_RELEASE} instead of calling them out... also... debug has bounds_check on??

Reply:
debug does bounds checking (fcheck=all)

Reply:
added for other compilers

Reply:
very nice... note that fbounds-check is a depreciated feature to be replaced with fcheck=bounds.